x/gov/README.md

```diff
@@ -737,22 +743,17 @@ The governance module emits the following events:
 
 The governance module contains the following parameters:
 
-| Key           | Type   | Example                                                                                            |
-|---------------|--------|----------------------------------------------------------------------------------------------------|
-| depositparams | object | {"min_deposit":[{"denom":"uatom","amount":"10000000"}],"max_deposit_period":"172800000000000"}     |
-| votingparams  | object | {"voting_period":"172800000000000"}                                                                |
-| tallyparams   | object | {"quorum":"0.334000000000000000","threshold":"0.500000000000000000","veto":"0.334000000000000000"} |
-
-### SubKeys
-
-| Key                | Type             | Example                                 |
-|--------------------|------------------|-----------------------------------------|
-| min_deposit        | array (coins)    | [{"denom":"uatom","amount":"10000000"}] |
-| max_deposit_period | string (time ns) | "172800000000000"                       |
-| voting_period      | string (time ns) | "172800000000000"                       |
-| quorum             | string (dec)     | "0.334000000000000000"                  |
-| threshold          | string (dec)     | "0.500000000000000000"                  |
-| veto               | string (dec)     | "0.334000000000000000"                  |
+| Key                     | Type             | Example                                 |
+| ----------------------- | ---------------- | --------------------------------------- |
+| min_deposit             | array (coins)    | [{"denom":"uatom","amount":"10000000"}] |
+| max_deposit_period      | string (time ns) | "172800000000000"                       |
+| voting_period           | string (time ns) | "172800000000000"                       |
+| quorum                  | string (dec)     | "0.334000000000000000"                  |
+| threshold               | string (dec)     | "0.500000000000000000"                  |
+| veto                    | string (dec)     | "0.334000000000000000"                  |
+| expedited_threshold     | string (time ns) | "0.667000000000000000"                  |
+| expedited_voting_period | string (time ns) | "86400000000000"                        |
```

Comment:
can we add a comment or () to say what is the time so people don't have to do the math? same with above